vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/PriorityByWorkflow.java

```diff
@@ -117,29 +101,31 @@ public synchronized ConsumableResourceResponse request(
               + "values should be one of the following: %s", workflowName, workflowPriority,
               acceptedPriorities.stream().map(String::valueOf).collect(Collectors.joining(", "))));
     }
+    if (workflows.get(workflowName) == null) {
```

Comment:
`set` adds the workflow to `workflows` if it doesn't yet exist, but `set` also uses your custom SortedSet logic to overwrite existing records with a new priority. If the server is requesting a workflow run we already know about, potentially with a new priority value, shouldn't this be where we update that?